pkg/cli/debug.go

```diff
@@ -528,19 +528,29 @@ func runDebugRangeDescriptors(cmd *cobra.Command, args []string) error {
 	})
 }
 
+var decodeKeyOptions struct {
+	useBase64 bool
+}
+
 var debugDecodeKeyCmd = &cobra.Command{
 	Use:   "decode-key",
 	Short: "decode <key>",
 	Long: `
-Decode a hexadecimal-encoded key and pretty-print it. For example:
+Decode a hexadecimal or base64 encoded key and pretty-print it. For example:
```

Comment:
Mention that hex is default and base64 needs the `--base64` flag.

Reply:
After some thinking I think using an enum flag hex|base64 would be better. It is more logical as we select format rather than override defaults.